common/control_files/contrail-vrouter-dpdk.ini

```diff
@@ -1,5 +1,5 @@
 [program:contrail-vrouter-dpdk]
-command=/usr/bin/contrail-vrouter-dpdk --no-daemon
+command=taskset __COREMASK__ /usr/bin/contrail-vrouter-dpdk --no-daemon
```

Comment:
This breaks the package. Vrouter will not start until I explicitly modify the *.ini file on the target system. What if I don't use the fabric scripts? What if I install the package manually on a target? It will not work.